core/ui/EditorToolbar/picture-dropdown.tid

```diff
@@ -4,8 +4,44 @@ title: $:/core/ui/EditorToolbar/picture-dropdown
 [img[$(imageTitle)$]]
 \end
 
+\define insert-external-image()
```

Comment:
As said, not sure if we're really aiming at external images here. I would think the extension should rather render the html for embedding image files outside of tiddlywiki, rather than to enter some title as if that was an image tiddler.

The thing is, we already know what image tiddlers are there, and that's what the current implementation relies on.

So, to me it would rather be helpful to filter upon those images via search terms..